src/object/srv_obj_migrate.c

```diff
@@ -1004,13 +1004,20 @@ migrate_dkey(struct migrate_pool_tls *tls, struct migrate_one *mrone)
 	data_size = daos_iods_len(mrone->mo_iods, mrone->mo_iod_num);
 
 	D_DEBUG(DB_TRACE, "data size is "DF_U64"\n", data_size);
+	D_PRINT("[RYON] %s:%d [%s()] > data_size: %lu\n", __FILE__, __LINE__, __FUNCTION__, data_size);
```

Comment:
(style)  line over 80 characters
(style)  __func__ should be used instead of gcc specific __FUNCTION__